src/main/resources/db/migration/V1_0_2__minimal_table.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE `service` CASCADE;
```

Comment:
I think, here it'll fail, if the table doesn't exist. 'IF EXISTS' would prevent an error

Reply:
The table is created in the previous db migration step, so i deemed it not necessary :sweat_smile:

Reply:
![sleepy-sloth](https://github.com/FIWARE/credentials-config-service/assets/24524518/59fa7afc-02bb-4701-88c2-ac6d7f10b272)